examples/js/controls/PanoramicControls.js

```diff
@@ -0,0 +1,229 @@
+THREE.PanoramicControls = function( camera, domElement )
+{
+	this.camera = ( camera !== undefined ) ? camera : console.error( "PanoramicControls required a THREE.PerspectiveCamera in first argument." );
+	this.domElement = ( domElement !== undefined ) ? domElement : document.body;
+	this.target = new THREE.Vector3();
+
+	this.autoRotateSpeed = 1.0;
+
+	this.minPolarAngle = 0.0;// degrees
```

Comment:
Orbit controls uses radians for this parameter. Do you mind using radians here for consistency?